src/main/assembly/dist/res/license/License.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
+        @page {
+            margin: 2.5cm 1.5cm 2cm 1.5cm;
+            size: A4;
+            @top-left {
+                content: "";
+                width: 100%;
+                height: 100%;
+                background: url(data:image/jpg;base64,$DansLogo) no-repeat 0 0;
+                background-position: left center;
+                background-size: 100px 30px;
+            }
+            @bottom-center {
+                content: "$FooterText - page " counter(page) "/" counter(pages);
+                font: 14px "Bitstream Charter";
+            }
+        }
+        body {
+            font: 14px "Bitstream Charter";
+            text-align: justify;
+        }
+        ol li ul {
+            list-style-type: disc;
+        }
+        ol {
+            list-style-type: lower-alpha;
+        }
+        table {
+            border-collapse: collapse;
+            border-spacing: 0;
+            display: table;
+            border: 1px solid black;
+            margin-left: auto;
+            margin-right: auto;
+        }
+        tr {
+            border-bottom: 1px solid black;
+        }
+        table td, table th {
+            padding: 6px 8px;
+            display: table-cell;
+            vertical-align: top;
+            border-right: 1px solid black;
+        }
+    </style>
+</head>
+<body>
+
+#parse("Body.html")
+
+#if ($OpenAccess)
+    #parse("OpenAccess.html")
+#elseif ($OpenAccessForRegisteredUsers)
+    #parse("OpenAccessForRegisteredUsers.html")
+#elseif ($OtherAccess)
+    #parse("OtherAccess.html")
+#elseif ($RestrictGroup)
+    #parse("RestrictGroup.html")
+#elseif ($RestrictRequest)
+    #parse("RestrictRequest.html")
+#else
+    <!-- default access right
+         never expect to be in this case, but I can't find an error function in Velocity -->
```

Comment:
I build this into the code rather than in the html. Seems better to me 😃 

I also added an error message in the pdf though. Was a good idea!